scripts/draft-blog-post.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+const changelogUnreleasedDir = path.join(__dirname, "../changelog_unreleased");
+const blogDir = path.join(__dirname, "../website/blog");
+
+const version = require("../package.json").version.replace(/-.+/, "");
+const versionShort = version.replace(/\.0$/, "");
+
+const categories = [
+  { dir: "javascript", title: "JavaScript" },
+  { dir: "typescript", title: "TypeScript" },
+  { dir: "flow", title: "Flow" },
+  { dir: "json", title: "JSON" },
+  { dir: "css", title: "CSS" },
+  { dir: "scss", title: "SCSS" },
+  { dir: "less", title: "Less" },
+  { dir: "html", title: "HTML" },
+  { dir: "vue", title: "Vue" },
+  { dir: "angular", title: "Angular" },
+  { dir: "lwc", title: "LWC" },
+  { dir: "handlebars", title: "Handlebars" },
+  { dir: "graphql", title: "GraphQL" },
+  { dir: "markdown", title: "Markdown" },
+  { dir: "mdx", title: "MDX" },
+  { dir: "yaml", title: "YAML" },
+  { dir: "api", title: "API" },
+  { dir: "cli", title: "CLI" }
+];
```

Comment:
Can we share this with `lint-changelog.js`? And I forgot add shebang on `lint-changelog.js`

Reply:
Yes, eventually it'll be shared, but so far I'm not sure where this script is going to end up. It might become part of the release script or else a page titled "Unreleased changes" might be generated every time the website is build. Or both.